addons/custom-block-shape/userscript.js

```diff
@@ -157,9 +158,9 @@ export default async function ({ addon, console }) {
       BlockSvg.SHAPE_IN_SHAPE_PADDING[1][2] = 5 * GRID_UNIT * multiplier;
       BlockSvg.SHAPE_IN_SHAPE_PADDING[1][3] = 5 * GRID_UNIT * multiplier;
 
-      var originalDropdownObject = BlocklyInstance.FieldDropdown.prototype.positionArrow;
       BlocklyInstance.FieldDropdown.prototype.positionArrow = function (x) {
-        this.arrowY_ = 11 * multiplier;
+        const arrowHeight = 12;
+        this.arrowY_ = (BlockSvg.FIELD_HEIGHT - arrowHeight) / 2 + 1;
```

Comment:
No problem if `BlockSvg.FIELD_HEIGHT - arrowHeight` is not even, right? It might end up being a decimal

Reply:
This value is used to set the element's `transform`, so decimals are fine.